Doc/library/turtle.rst

```diff
@@ -1275,6 +1303,29 @@ Filling
       ... else:
       ...    turtle.pensize(3)
 
+.. function:: fill()
```

Comment:
This is more of an aside, apparently there was a `fill()` in Python 2.5 and earlier:

> ## Changes since Python 2.6
> ...
> The method Turtle.fill() has been eliminated. The behaviour of [begin_fill()](https://docs.python.org/3/library/turtle.html#turtle.begin_fill) and [end_fill()](https://docs.python.org/3/library/turtle.html#turtle.end_fill) have changed slightly: now every filling process must be completed with an end_fill() call.

https://docs.python.org/3/library/turtle.html#changes-since-python-2-6

I don't think we need to worry about people switching between 2.5 (EOL in 2011) and 3.14 (to be released in 2025) and getting confused.